docs/guide/service/annotations.md

```diff
@@ -416,3 +419,6 @@ Load balancer access can be controlled via following annotations:
 The AWS Load Balancer Controller manages Kubernetes Services in a compatible way with the legacy aws cloud provider. The annotation `service.beta.kubernetes.io/aws-load-balancer-type` is used to determine which controller reconciles the service. If the annotation value is `nlb-ip` or `external`, legacy cloud provider ignores the service resource (provided it has the correct patch) so that the AWS Load Balancer controller can take over. For all other values of the annotation, the legacy cloud provider will handle the service. Note that this annotation should be specified during service creation and not edited later.
 
 The legacy cloud provider patch was added in Kubernetes v1.20 and is backported to Kubernetes v1.18.18+, v1.19.10+.
+
+!!!note ""
+    With the right combination of annotations any Network Load Balancer and Target Group configuration can be provisioned with this controller.
```

Comment:
This note doesn't provide additional information. Lets remove it.

Reply:
> The AWS Load Balancer Controller manages Kubernetes Services in a compatible way with the legacy aws cloud provider. The annotation `service.beta.kubernetes.io/aws-load-balancer-type` is used to determine which controller reconciles the service. If the annotation value is `nlb-ip` or `external`, legacy cloud provider ignores the service resource (provided it has the correct patch) so that the AWS Load Balancer controller can take over. For all other values of the annotation, the legacy cloud provider will handle the service. Note that this annotation should be specified during service creation and not edited later.

As with the main issue, right after reading that paragraph, I assumed that the new controller can not manage an internal load balancer with instance target type. If you are not okay adding that note, maybe I could rewrite that paragraph to make that clearer?

Reply:
sounds better